source/includes/_organizations.md

```diff
@@ -59,6 +68,9 @@ In order to interact with this endpoint you must pass an admin API key. Visit [o
 |`org_contact_name`|String|*Required*. The individual who can be contacted with any questions about the organization.|
 |`org_email`|String|*Required*. The email address by which the organization may be contacted.|
 |`org_url`|String|The URL for the organization's website.|
+|`external_admin_group`|String|The name of the admin group from the external identity system when using external organization membership management.|
+|`external_member_group`|String|The name of the member group from the external identity system when using external organization membership management.|
+|`external_id`|String|The identifier for this organization in the external identity system when using external organization membership management.|
```

Comment:
We can update this to be more general I think - something like "A unique identifier that can be used to synchronize algorithmia organizations with external systems, such as an identity provider"